src/managing-workflow/production-deployments.md

```diff
@@ -1,3 +1,57 @@
 # Production Deployments
 
-TODO (bacongobbler): rewrite for v2
+Many Workflow users are running Workflow quite successfully in production. When readying a Workflow
```

Comment:
I assume this is a forward-looking statement. Still, I would drop the first sentence completely until such time as it becomes accurate.